sonic_platform_base/sonic_xcvr/api/public/cmis.py

```diff
@@ -407,55 +398,55 @@ def get_transceiver_dom_flags(self):
             case_temp_flags = module_flag['case_temp_flags']
             voltage_flags = module_flag['voltage_flags']
             dom_flag_dict.update({
-                'temphighalarm': case_temp_flags['case_temp_high_alarm_flag'],
-                'templowalarm': case_temp_flags['case_temp_low_alarm_flag'],
-                'temphighwarning': case_temp_flags['case_temp_high_warn_flag'],
-                'templowwarning': case_temp_flags['case_temp_low_warn_flag'],
-                'vcchighalarm': voltage_flags['voltage_high_alarm_flag'],
-                'vcclowalarm': voltage_flags['voltage_low_alarm_flag'],
-                'vcchighwarning': voltage_flags['voltage_high_warn_flag'],
-                'vcclowwarning': voltage_flags['voltage_low_warn_flag']
+                'tempHAlarm': case_temp_flags['case_temp_high_alarm_flag'],
```

Comment:
@mihirpat1 can we rename get_transceiver_bulk_status to get_dom_sensor() and state that this will fetch real time dom sensor number value. get_transceiver_dom_flags() will only get flag T/F values. Please make this change for SFF8436, SFF8636 as well

Reply:
@prgeor I have addressed this now